Applications/Games/The Sims 3/Local/script.js

```diff
@@ -13,7 +13,7 @@ var installerImplementation = {
             .author("Zemogiter")
             .category("Games")
             .executable("Sims3Launcher.exe", ["xgamma -gamma 1"])
-            .wineVersion("3.19")
+            .wineVersion("4.0.1")
```

Comment:
Wouldn't it make sense to use the default (latest stable)?

Reply:
Maybe but what if they release a new stable 3 months from now and the script stops working?

Reply:
That can always happen. On the other hand, with this approach, somebody always must update the versions manually which is worse in my opinion.

Reply:
You have a point. I guess lack of action is worse than any action.